src/components/radio/radio-button.scss

```diff
@@ -0,0 +1,139 @@
+@import "default-theme";
+
+$radio-width: 20px !default;
+$radio-height: $radio-width !default;
+
+// Top-level container for entire button.
+md-radio-button {
+  display: inline-block;
+  cursor: pointer;
+  padding: 8px;
+  position: relative;
+  white-space: nowrap;
+}
+
+// Container for radio circles and ripple.
+.md-radio-container {
+  box-sizing: border-box;
+  display: inline-block;
+  height: $radio-width;
+  position: relative;
+  top: 2px;
+  width: $radio-width;
+}
+
+// TODO(mtlin): Replace when ink ripple component is implemented.
+// A placeholder ink ripple, shown when keyboard focused.
+.md-ink-ripple {
+  background: #000;
+  border-radius: 50%;
+  height: 48px;
+  left: 10px;
+  opacity: 0;
+  pointer-events: none;
+  position: absolute;
+  top: 10px;
+  transform: translate(-50%,-50%);
+  transition: opacity ease 0.28s, background-color ease 0.28s;
+  width: 48px;
+  overflow: hidden;
+}
+
+// The outer circle for the radio, always present.
+.md-radio-outer-circle {
+  border: solid 2px;
+  border-radius: 50%;
+  box-sizing: border-box;
+  height: $radio-width;
+  left: 0;
+  position: absolute;
+  top: 0;
+  transition: border-color ease 0.28s;
+  width: $radio-width;
+}
+
+// The inner circle for the radio, shown when checked.
+.md-radio-inner-circle {
+  border-radius: 50%;
+  box-sizing: border-box;
+  height: $radio-width;
+  left: 0;
+  position: absolute;
+  top: 0;
+  transition: transform ease 0.28s, background-color ease 0.28s;
```

Comment:
Potentially, but I am holding off on any design/motion tweaks because I need to get those reviewed by the team first.